build.gradle

```diff
@@ -46,7 +46,7 @@ task coverage(type: JacocoReport) {
 
 dependencies {
     String jUnitVersion = '5.4.0'
-    String javaFxVersion = '11'
+    String javaFxVersion = '11.0.1'
```

Comment:
Hmm why change to 11.0.1?

Reply:
it was showing a runtime warning that it running 11.0.1 when we only using 11